### PR TITLE
Set up and mobile first

### PR DESCRIPTION
**Pull request by dumbledorep**
### General requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
    - Linter should be set up correctly for this repository.
    - Linter result should be green.
    - There should be no custom changes to linter config.
- Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
    - README file should be descriptive.
    - Commit messages should be meaningful.
    - Pull Request should have a descriptive title explaining what changes are inside.
    - Pull Request should have a short summary describing introduced changes in more detail.
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
    - Branch name should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
    - Pull Request is made from 'feature branch' into the `main` branch.

### HTML/CSS requirements

- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### Project requirements
- For this project, you should use the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- The 2 sections you need to build for this project are:

    - the toolbar (or header):
 
    - the headline section (right after the header):
   

  
We included a screenshot of Template 1 for reference, but you should follow the template that you chose.
    
- In order to lay out the elements on the page you should use Flexbox in all 2 sections.
- You don't need to implement any functionality that requires JavaScript, like opening the menu or creating dropdowns.
- You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

**Comment by smunozmo** [comment]
Hi @dumbledorep, 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

- You have submitted a Pull Request that has no description, please add a description and submit it again for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/dumbledorep/Phil-s-Portfolio#readme",
   "devDependencies": {
-    "hint": "^7.0.1",
+    "hint": "^7.1.0",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
```
Comment (MaganaAsiati) [inline package.json:23]: kindly make sure your page is responsive and match the design. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @dumbledorep  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

170491251-b1add8ba-b18f-4648-a893-20e510d957fd


## Highlights
✅ No linter error
✅ README is professional
✅ correct use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 2]
```diff
@@ -30,12 +30,6 @@
           <div><a href="#about">About</a></div>
           <div><a href="#contact">Contact</a></div>
```
Comment (mgmediaweb) [inline index.html:31]: Kindly, the main menu should be a UL/LI listing

**Review by mgmediaweb — APPROVED** [review]
Hi @dumbledorep , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

[OPTIONAL] Kindly, for these exercises we will use Media Queries, so I recommend you to use only 1 style file and within it use Media Queries.

[OPTIONAL] The navbar menu should be UL/LI listed, it would be nice if you made that change.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**